### PR TITLE
hotfix: discussions bugs

### PR DESCRIPTION
**Pull request by elbotho**
- threads can now be collapsed again
- scrolling is deactivated on `/discussions` page

for https://serlo.slack.com/archives/C0BMSC431/p1666617395168269

### CODE REVIEW

File: src/components/comments/comment-area.tsx [hunk 1]
```diff
@@ -56,6 +57,10 @@ export function CommentArea({
       window.location.hash.startsWith('#comment-')) ||
     highlightedCommentId !== undefined
 
+  const [showThreadChildren, setShowThreadChildren] = useState<string[]>(
```
Comment (kulla) [inline src/components/comments/comment-area.tsx:60]: Why is a state necessary here?

**Review by kulla — APPROVED** [review]

File: src/components/comments/comment-area.tsx (hunk 1 above)
Comment (elbotho) [reply src/components/comments/comment-area.tsx:60]: because some users really like the option to toggle the visibility of the comments in one thread

Comment (kulla) [reply src/components/comments/comment-area.tsx:60]: I ask because `setShowThreadChildren` is nowhere used. Just a comment though -> no blocker :smile:

Comment (elbotho) [reply src/components/comments/comment-area.tsx:60]: Alright, It's used in this file, so we are good :)

**Pull request merged**